picard/metadata.py

```diff
@@ -150,10 +150,13 @@ def compare_to_release_parts(self, release, weights):
         except (ValueError, KeyError):
             pass
         else:
-            if "title" in weights:
-                b = release['media'][0]['track-count']
-            else:
-                b = release['track-count']
+            try:
+                if "title" in weights:
+                    b = release['media'][0]['track-count']
+                else:
+                    b = release['track-count']
+            except KeyError:
```

Comment:
Could this also raise an IndexError?

Reply:
I don't think so. If the release contains a media, it must have a first element, and if it doesn't contain a media then it will already raise a KeyError when evaluating release['media']. I don't think there's such a case as release['media'] being an empty list. Or there is?